sklearn/feature_selection/_rfe.py

```diff
@@ -667,7 +674,17 @@ class RFECV(RFE):
         n_features : ndarray of shape (n_subsets_of_features,)
             Number of features used at each step.
 
-            .. versionadded:: 1.5
+          .. versionadded:: 1.5
+
+        split(k)_ranking : ndarray of shape (n_subsets_of_features,)
+            The cross-validation rankings across (k)th fold.
+
+          .. versionadded:: 1.7
+
+        split(k)_support : ndarray of shape (n_subsets_of_features,)
+            The cross-validation supports across (k)th fold.
+
+          .. versionadded:: 1.7
```

Comment:
can we have links to a part of the user guide or an example where "ranking" and "support" are nicely explained? I don't find the intuitive enough for somebody who reads this the first time.

Reply:
I found something for the ranking, but nothing for the support. I enriched the description.